test/syscall/syscall.py

```diff
@@ -628,7 +628,12 @@ def run(self):
         errfile = open(self.errfilename, 'w')
         if self.args.verbose:
             print('RUNNING: ' + str(callargs))
-        subret = subprocess.call(callargs, stdout=outfile, stderr=errfile)
+        # We have to overload the WIREDTIGER_CONFIG to ensure we perform
+        # fdatasync operations on checkpoints
+        my_env = os.environ.copy()
+        my_env["WIREDTIGER_CONFIG"] = "checkpoint_sync=true"
```

Comment:
I'd recommend just `del my_env["WIREDTIGER_CONFIG"]` here -- we should test the defaults, it's only because we run this test with Jenkins and in most other cases want to avoid hammering disks unnecessarily.